core/mixins.py

```diff
@@ -16,6 +16,12 @@ def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
 
+class CampaignPagesFeatureFlagMixin(NotFoundOnDisabledFeature):
+    @property
+    def flag(self):
+        return settings.FEATURE_FLAGS['CAMPAIGN_PAGES_ON']
```

Comment:
I was going to suggest that as this is not a dynamic value we can do:

```
class CampaignPagesFeatureFlagMixin(NotFoundOnDisabledFeature):
    fag = settings.FEATURE_FLAGS['CAMPAIGN_PAGES_ON']
```

**but** your way is easier to test :+1:

Reply:
don't look at me i just copypasted one of the other feature flag mixins :'D